header/helper.go

```diff
@@ -0,0 +1,36 @@
+package header
+
+import (
+	"bytes"
+	"sort"
+)
+
+// GetFrequentHeader chooses the most frequent header.
+// NOTE: if slice does not have duplicates, then GetFrequentHeader returns the first entry of the passed slice.
+func GetFrequentHeader(headers []*ExtendedHeader) *ExtendedHeader {
```

Comment:
REQ: This function needs a unit test.

Reply:
Conditions were changed. This approach is not actual anymore

Reply:
n/a